library/src/main/java/io/appwrite/services/Avatars.kt

```diff
@@ -92,7 +92,7 @@ class Avatars(client: Client) : Service(client) {
     /**
      * Get favicon
      *
-     * Use this endpoint to fetch the favorite icon (AKA favicon) of any remote website URL.
+     * Use this endpoint to fetch the favorite icon (AKA favicon) of any remote website URL.This endpoint does not follow HTTP redirects.
```

Comment:
Newlines are missing before all `This endpoint does not follow HTTP redirects` instances

Reply:
![Screenshot 2024-08-27 at 3 20 42 PM](https://github.com/user-attachments/assets/1aec8d13-34d3-4144-89d1-3b940476164f)

As I can see the MD files contain the new line. But I guess twig strips them. I believe this is not a blocker right?